embeddings/pipeline/flair_sequence_labeling.py

```diff
@@ -59,7 +61,11 @@ def __init__(
                 SampleSplitsFlairCorpusTransformation(*sample_missing_splits, seed=seed)
             )
 
-        embedding = AutoFlairWordEmbedding.from_hub(embedding_name)
+        if isinstance(embedding_name, pathlib.Path):
```

Comment:
I would add that functionality to `AutoFlairWordEmbedding`. Instead of here. Error with loading embeddings is internal problem with flair itself, therefore we shouldn't use that internally in pipelines but I would wrap this in `AutoFlairWordEmbedding`